app/uk/gov/nationalarchives/omega/editorial/controllers/EditSetController.scala

```diff
@@ -81,12 +82,14 @@ class EditSetController @Inject() (val messagesControllerComponents: MessagesCon
     * a path of `/edit-set/{id}`.
     */
   def view(id: String) = Action { implicit request: Request[AnyContent] =>
-    logger.info(s"The edit set id is $id ")
-    val editSet = getEditSet(id)
-    val messages: Messages = request.messages
-    val title: String = messages("edit-set.title")
-    val heading: String = messages("edit-set.heading", editSet.name)
-    Ok(views.html.editSet(title, heading, editSet))
+    withUser { user =>
```

Comment:
I am seeing compiler warnings because `user` is not being used. Replacing `user` with `_` will prevent these warnings.